scalalib/worker/src/ZincWorkerImpl.scala

```diff
@@ -411,6 +411,11 @@ class ZincWorkerImpl(compilerBridge: Either[
           r.logInfo(new ZincProblem(problem))
           super.logInfo(problem)
         }
+
+        override def printSummary(): Unit = {
+          r.printSummary()
+          super.printSummary()
+        }
```

Comment:
That the override that was missing to trigger the finish compilation notification